OAuth/ResourceOwner/Auth0ResourceOwner.php

```diff
@@ -34,19 +34,48 @@ class Auth0ResourceOwner extends GenericOAuth2ResourceOwner
         'profilepicture' => 'picture',
     );
 
+    protected function getAuthClient() {
```

Comment:
Hmmm, do you want to overwrite those new methods in your code? If no, I would remove this whole method and move code into `configureOptions()`.